### PR TITLE
Che ping/kinmen/feat dr bid page prototype

### PR DESCRIPTION
**Pull request by curry0622**
- Create fake data
- User is able to choose a date to change data
- `Status` showing total dr volume and total price
- List prototype
- RWD done
- i18n done

### CODE REVIEW

File: src/components/drBidPage/submit/typeButton.tsx [hunk 1]
```diff
@@ -26,28 +26,28 @@ const TypeButton: React.FC<IProps> = ({ setType }) => {
...
-          'bidding-submit-modebutton-button',
-          'bidding-submit-modebutton-button-buy',
+          'drbid-submit-modebutton-button',
+          'drbid-submit-modebutton-button-buy',
         )}
         disabled={disabled}
         onClick={() => handleClickBuy()}
       >
-        {t('biddingpage.buy')}
+        {t('drbidpage.day')}
```
Comment (coliver0221) [inline src/components/drBidPage/submit/typeButton.tsx]: I think `.dayAhead` is more clear to understand.

**Review by coliver0221 — CHANGES_REQUESTED** [review]
I think it's not necessary to separate `drBidPage/status` into main and sub components since there is only one sub-component.
It's fine to just have `drBidPage/status/index.tsx`.

File: src/components/drBidPage/submit/typeButton.tsx (hunk 1 above)
Comment (curry0622) [reply src/components/drBidPage/submit/typeButton.tsx]: Done. Thx

**Comment by coliver0221** [comment]
Please fix the conflict, thanks.

**Review by coliver0221 — APPROVED** [review]

**Comment by coliver0221** [comment]
LGTM, thanks.

**Pull request merged**